### PR TITLE
C++: tidy up

### PR DESCRIPTION
**Pull request by jmillan**

### CODE REVIEW

File: worker/src/RTC/PortManager.cpp [hunk 1]
```diff
@@ -47,15 +47,17 @@ namespace RTC
 
 		int err;
 		const int family = Utils::IP::GetFamily(ip);
-		struct sockaddr_storage bindAddr; // NOLINT(cppcoreguidelines-pro-type-member-init)
+		struct sockaddr_storage bindAddr
+		{
+		}; // NOLINT
```
Comment (jmillan) [inline worker/src/RTC/PortManager.cpp]: For some reason I didn't find, clang-format adds a new line inside `{}`

File: worker/src/RTC/RtpRetransmissionBuffer.cpp [hunk 2]
```diff
@@ -7,6 +7,37 @@
 
 namespace RTC
 {
+	/* Class methods. */
+
+	RtpRetransmissionBuffer::Item* RtpRetransmissionBuffer::FillItem(
```
Comment (jmillan) [inline worker/src/RTC/RtpRetransmissionBuffer.cpp:12]: made static.

File: worker/include/RTC/PlainTransport.hpp [hunk 3]
```diff
@@ -76,8 +76,12 @@ namespace RTC
 		ListenInfo rtcpListenInfo;
 		bool rtcpMux{ true };
 		bool comedia{ false };
-		struct sockaddr_storage remoteAddrStorage;
-		struct sockaddr_storage rtcpRemoteAddrStorage;
+		struct sockaddr_storage remoteAddrStorage
+		{
+		}; // NOLINT
```
Comment (ibc) [inline worker/include/RTC/PlainTransport.hpp]: This NOLINT is intended for lint or for tidy task?

File: worker/src/RTC/SctpAssociation.cpp [hunk 4]
```diff
@@ -115,29 +115,25 @@ namespace RTC
...
-		// Get a id for this SctpAssociation.
-		this->id = DepUsrSCTP::GetNextSctpAssociationId();
-
 		// Register ourselves in usrsctp.
 		// NOTE: This must be done before calling usrsctp_bind().
 		usrsctp_register_address(reinterpret_cast<void*>(this->id));
 
 		int ret;
 
-		this->socket = usrsctp_socket(
```
Comment (jmillan) [inline worker/src/RTC/SctpAssociation.cpp]: I'm not big fan of initializing `socket` in above.., we can add a // NOLINT here and bring it back where it was.

**Review by ibc — COMMENTED** [review]
Can you comment a bit about how to use this? In which environment does it work? Only in Linux/Docker? Requirements? Do we have to install deps at global level or something? If so, which versions?

File: worker/src/RTC/WebRtcServer.cpp [hunk 5]
```diff
@@ -25,6 +25,30 @@ namespace RTC
 		       std::pow(2, 0) * (256 - IceComponent);
 	}
 
+	/* Class methods. */
+
+	inline std::string WebRtcServer::GetLocalIceUsernameFragmentFromReceivedStunPacket(
```
Comment (jmillan) [inline worker/src/RTC/WebRtcServer.cpp:30]: Made static.

**Comment by ibc** [comment]
CI failing

**Comment by jmillan** [comment]
> Can you comment a bit about how to use this? In which environment does it work? Only in Linux/Docker? Requirements? Do we have to install deps at global level or something? If so, which versions?

These are just code changes. The `lint` command is not YET uploaded, and even code changes have not finished. I'm doing those in different PRs. I'm going step by step. This is not as automated as it seems yet.

**Comment by ibc** [comment]
> These are just code changes. The `lint` command is not YET be uploaded, and even code changes have not finished. I'm doing those in different PRs. I'm going step by step. This is not as automated as it seems yet.

Ok, so what is the command to make these errors show up? `invoke tify`?

File: worker/src/RTC/SctpAssociation.cpp (hunk 4 above)
Comment (ibc) [reply worker/src/RTC/SctpAssociation.cpp]: I agree. If such an initialization of `socket` fails we would not even see the `MS_TRACE`.

**Comment by jmillan** [comment]
> Ok, so what is the command to make these errors show up? invoke tify?

Yes, yes, how we call tidy will not change.

File: worker/include/RTC/PlainTransport.hpp (hunk 3 above)
Comment (jmillan) [reply worker/include/RTC/PlainTransport.hpp]: These where indeed no needed. Removed.

File: worker/src/RTC/SctpAssociation.cpp (hunk 4 above)
Comment (jmillan) [reply worker/src/RTC/SctpAssociation.cpp]: Done.

I'll add the corresponding // NOLINT directive in a following PR.

File: worker/src/RTC/PortManager.cpp (hunk 1 above)
Comment (jmillan) [reply worker/src/RTC/PortManager.cpp]: This is because we force brace in new line for structs. Makes sense.

**Comment by jmillan** [comment]
@ibc, CI passing and ready to merge. Let  me know if you have any concern so I'll continue with the next PR.

**Review by ibc — APPROVED** [review]

**Comment by ibc** [comment]
Approved.

**Pull request merged**